deploy/helm/metrics-collector-config.yaml

```diff
@@ -1004,6 +1023,21 @@ receivers:
       grpc:
         endpoint: 0.0.0.0:{{ .Values.otel.metrics.otlp_endpoint.port }}
   k8s_events:
+{{- if .Values.opencost.enabled }}
+  prometheus/opencost:
```

Comment:
Does this prom "for opencost" needs to scrape opencost itself? Why?

Reply:
how would you scrape it? as application?